html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+    <img src="https://cdnb.artstation.com/p/assets/images/images/024/574/333/4k/mickael-lelievre-zubat-d.jpg?1582844350&dl=1" alt="Pokemon Zubat" width="600px" height="600px"/>
+    <h1>Zubat</h1>
+    <p>&copy Pokemon.&reg Nintendo.</p>
```

Comment:
Make sure to close your HTML entities with a `;`

Reply:
![html-syntax](https://user-images.githubusercontent.com/25681013/135951645-f83628d6-10a6-4130-bf88-cf47e3363246.png)